src/components/LoansByCategory/HelpmeChoose/HelpmeChooseRecommendations.vue

```diff
@@ -125,6 +153,13 @@ export default {
 		},
 		getImageUrl(loan) {
 			return loan?.image?.default ?? '';
+		},
+		handleAddToBasket() {
+			this.$kvTrackEvent(
```

Comment:
This event should be ['loan-card' 'add-to-basket', null, <loan id>]

If you really need an identifier for help me choose, then you can use:
 ['loan-card' 'add-to-basket', 'help-choosing', <loan id>]